src/handlers/payout/post.ts

```diff
@@ -34,15 +34,26 @@ export const incentivizeComments = async () => {
     return;
   }
 
+  const tokenSymbol = await getTokenSymbol(paymentToken, rpc);
+
   const issueComments = await getAllIssueComments(payload.issue?.number!);
   logger.info(`Getting the issue comments done. comments: ${JSON.stringify(issueComments)}`);
+
+  const [creatorComment] = issueComments.splice(0, 1);
+  const creator = creatorComment.user;
+  if (!(creator.type == UserType.Bot || creator.login == assignee)) {
+    const comments = [creatorComment.body];
+    const result = await generatePermitForCommit(creator.login, comments, issueCreatorMultiplier, commentElementPricing, tokenSymbol, issue.node_id);
+    if (result[0]) logger.info(`Permit url generated for creator. reward: ${result[0]}`);
+    if (result[1]) logger.info(`Skipping to generate a premit url for missing creator. fallback:${result[1]}`);
+  }
+
```

Comment:
This is not resolved yet. Let me clarify things for you.

1/ Please never update the codebase of `incentivizeComments`. because its connected to other working PR and not completed yet.
2/ Create a new function in `post.ts` like you recently created `incentivizeCreatorComments`.
3/ Add a `incentivizeCreatorComments` functions to the post processors [here](https://github.com/wannacfuture/ubiquibot/blob/feat/adjust-payouts-comment-incentives/src/handlers/processors.ts#L46)